### PR TITLE
chore(build): check for missing copyright messages

### PR DESCRIPTION
**Pull request by kmova**
Add a build-check to catch for missing copyright messages. This PR also adds the missing copyright messages. The commits are split into two:
- Update the source files with copyright message
- Add the automated check as part of the travis build.

### CODE REVIEW

File: buildscripts/Makefile.common [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright © 2019 The OpenEBS Authors
```
Comment (AmitKumarDas) [inline buildscripts/Makefile.common]: Is it possible to have this file suffixed with something that declares this file as a makefile

File: cmd/cstor-pool-mgmt/app/command/commands_test.go [hunk 2]
```diff
@@ -1,3 +1,17 @@
+// Copyright © 2017-2019 The OpenEBS Authors
```
Comment (prateekpandey14) [inline cmd/cstor-pool-mgmt/app/command/commands_test.go]: Can we just have 2017 here @kmova if possible, as i seen in other projects as well( kubernetes etc..), they mentioned only the year when the file has been created.

Comment (kmova) [reply cmd/cstor-pool-mgmt/app/command/commands_test.go]: Good point! we need to automate the validation of dates as well.

File: buildscripts/Makefile.common (hunk 1 above)
Comment (kmova) [reply buildscripts/Makefile.common]: renamed to common.mk.

**Pull request merged**